rescheduler/rescheduler.go

```diff
@@ -371,23 +375,19 @@ func prepareNodeForPod(client kube_client.Interface, recorder kube_record.EventR
 	return nil
 }
 
-func copyNode(node *apiv1.Node) (*apiv1.Node, error) {
-	objCopy, err := api.Scheme.DeepCopy(node)
-	if err != nil {
-		return nil, err
-	}
-	copied, ok := objCopy.(*apiv1.Node)
+func copyNode(obj interface{}) (*v1.Node, error) {
+	copied, ok := obj.(*v1.Node)
```

Comment:
Why did you remove DeepCopy? The object is not copied anymore.

Reply:
Yes, you are right. I missed that part. I wanted to do a `node.DeepCopy()` and remove this function.